### PR TITLE
add indices for work and flatedge

### PR DESCRIPTION
**Pull request by yetti**
@jrixon @scoen @shuangzhou @ninhnguyen @weijunnla @m-r-c

### CODE REVIEW

File: AmberDBv2MigrationScript.sql [hunk 1]
```diff
@@ -573,6 +575,8 @@ insert into work select * from work_history where txn_end = 0;
                                  
 CREATE INDEX work_id ON work (id);
 CREATE INDEX work_txn_id ON work (id, txn_start, txn_end);
+CREATE INDEX work_history_restricted_child ON work_history (accessConditions, bibLevel);
```
Comment (shuangzhou) [inline AmberDBv2MigrationScript.sql]: might need to update the table and index name for this index.

File: AmberDBv2MigrationScript.sql [hunk 2]
```diff
@@ -573,6 +575,8 @@ insert into work select * from work_history where txn_end = 0;
...
+CREATE INDEX work_digitalStatus ON work_history (digitalStatus);
```
Comment (shuangzhou) [inline AmberDBv2MigrationScript.sql]: might need to update the table name to work.

File: AmberDBv2MigrationScript.sql [hunk 3]
```diff
@@ -740,6 +744,8 @@ CREATE TABLE IF NOT EXISTS sess_work (
                                  workPid                                    TEXT) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
 CREATE INDEX sess_work_id ON sess_work (id);
 CREATE INDEX sess_work_txn_id ON sess_work (id, txn_start, txn_end);
+CREATE INDEX work_history_restricted_child ON work_history (accessConditions, bibLevel);
+CREATE INDEX sess_work_digitalStatus ON work_history (digitalStatus);
```
Comment (shuangzhou) [inline AmberDBv2MigrationScript.sql]: the above comments apply here also.

**Comment by jrixon** [comment]
🏁   🐎

**Comment by shuangzhou** [comment]
👍

**Comment by yetti** [comment]
@shuangzhou just letting you know, I did fix the errors in master after this PR was merged: https://github.com/nla/amberdb/commit/16d556e1a9558c65fa2c31f7be507814f1212da5

**Pull request merged**